packages/merkle-distributor/scripts/1_CreateClaimsForWindow.ts

```diff
@@ -0,0 +1,84 @@
+// Read in a json file containing information for a particular window distribution and produce a claims file containing
+// merkle proofs and a merkle root. This input file is produced from the liquidity/developer/KPI outputs output scripts.
+// The output of this script is then fed into the PublishClaimsForWindow.ts script to add claims to IPFS and Ethereum.
+
+// The input file should implement the following formatting:
+// {
+// "chainId": 42,
+// "rewardToken": "0x47B1EE6d02af0AA5082C90Ea1c2c14c70399186c",
+// "windowIndex": 0,
```

Comment:
The window is double-checked to match what is expected in the second script. it will error out if it is not what is expected.

Reply:
it totally could be. it can also be pulled from the contract. Right now I included it to ensure consistency between our expected index and the true index. in the second script, the on-chain index is pulled and compared to the file and triple checked.